src/modal_for_noobs/complete_dashboard.py

```
@@ -873,24 +1053,43 @@ def generate_enhanced_deployment(
         
         return dashboard
     
-    def _get_auth_status_html(self, is_authenticated: bool, status: str = None, workspace: str = None, error_msg: str = None) -> str:
+    def _get_auth_status_html(self, is_authenticated: bool, status: str = None, auth_type: str = None, error_msg: str = None) -> str:
```

Ruff (0.11.9)

PEP 484 prohibits implicit `Optional`

(RUF013)